airflow/contrib/auth/backends/github_enterprise_auth.py

```diff
@@ -84,17 +84,18 @@ def __init__(self):
         self.login_manager.login_view = 'airflow.login'
         self.flask_app = None
         self.ghe_oauth = None
-        self.api_rev = None
+        self.api_url = None
 
     def ghe_api_route(self, leaf):
-        if not self.api_rev:
-            self.api_rev = get_config_param('api_rev')
-
-        return '/'.join(['https:/',
-                         self.ghe_host,
-                         'api',
-                         self.api_rev,
-                         leaf.strip('/')])
+        if not self.api_url:
+            self.api_url = (
+                'https://api.github.com' if self.ghe_host == 'github.com'
+                else '/'.join(['https:/',
+                               self.ghe_host,
+                               'api',
+                               get_config_param('api_rev')])
+            )
+        return self.api_url + leaf
```

Comment:
hmm, one more thing,  should there be a '/' between the api_url and the leaf? could you also add unit test for this? thanks!

Reply:
Sure, I will! This method is used AFAIK only in this class and the leaf argument is passed with a '/'.
Do you think I should support also the case in which the '/' is missing?
Thanks for the feedback!

Reply:
Ah sorry, the diff view didn't show that far down. No big deal then.

Reply:
Could you maybe add a note about this in the security.rst file (after the GitHub Enterprise (GHE) Authentication section). This auth back end was originally only used for GHE. Now that you've extended the functionality for github.com, may be worth noting in the doc.

Reply:
@jgao54 just added a quick explanation in `security.rst` as you suggested.

Reply:
@renzofrigato  thanks!

Reply:
thank you for the review!